reporters/std-json-reporter.js

```diff
@@ -0,0 +1,30 @@
+const { StdReporter } = require('./std-reporter')
+
+class StdJsonReporter extends StdReporter {
+  format ({ message }) {
+    return JSON.stringify(message, null, 2)
+  }
+
+  logResults ({ results }) {
+    let output = results.map(result => ({
+      name: result.file.name,
+      passed: !result.failed,
+      size: result.file.size,
+      loading: result.file.loading,
+      running: result.file.running
+    }))
+    super.log({
+      message: this.format({ message: output })
+    })
+  }
+
+  error ({ message }) {
+    super.error({
+      message: this.format({ message: { error: message } })
+    })
+  }
+}
+
+module.exports = {
```

Comment:
We can directly export class here

Reply:
Ok. 
Do you think that named export to save naming consistence in this case is bad practice?

Reply:
I am exporting function directly without object wrap https://github.com/ai/size-limit/blob/master/index.js#L255

So for consistency, we should do the same with new files.

Reply:
Done for all new modules